### PR TITLE
Update for TLS 1.3 and pentest pages v platform

### PR DESCRIPTION
**Pull request by pburkholder**
## Changes proposed in this pull request:
-
- Note TLS 1.3 is coming
- Clarifiy pentest for pages
-

:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/peterb/tls-and-pentest-tweaks)

## Security Considerations
Clarifications and updates unrelated to operational security. Safe.

### CODE REVIEW

File: _docs/compliance/pentest.md [hunk 1]
```diff
@@ -18,4 +18,14 @@ If you, or a third party acting on your behalf, plan to perform a penetration te
 
 This notification is only necessary for in-depth security testing or significant load-testing, which is a common step in agency ATO processes for customer systems and in the software development lifecycle. You don't need an approval, and cloud.gov doesn't provide approvals. Simply sending the notification is sufficient. You can always run routine automated vulnerability scans on your own applications without special notification.
 
-When arranging a security assessment or penetration test, the system under test is _your_ application at _application-name_.app.cloud.gov, not the cloud.gov infrastructure -- as cloud.gov, and third-party assessors, test those continually and [make the results available through FedRAMP]({{ site.baseurl }}/docs/overview/fedramp-tracker/#start-the-ato-process). Your assessment must not target other cloud.gov customers, nor perform or simulate denial of service attacks or otherwise violate the [Amazon AWS testing policy](https://aws.amazon.com/security/penetration-testing/).
+When arranging a security assessment or penetration test, the system under test is one of:
+
+* _your_ application at _application-name_.app.cloud.gov, for cloud.gov Platform, or
```
Comment (jameshochadel) [inline _docs/compliance/pentest.md]: What about their external domain? Is there a reason they wouldn't be allowed to test against that?

File: _docs/compliance/pentest.md [hunk 2]
```diff
@@ -18,4 +18,14 @@ If you, or a third party acting on your behalf, plan to perform a penetration te
...
+* _your_ website at your preview URL, or your production website, for cloud.gov Pages.
+
+In either case, do not test the cloud.gov infrastructure, such as https://pages.cloud.gov or https://dashboard.fr.cloud.gov. All cloud.gov products are under regular testing by our team, and by third-party assessors, as part of our Continuous Monitoring plan. FedRAMP® [makes the results available]({{ site.baseurl }}/docs/overview/fedramp-tracker/#start-the-ato-process) to authorized users.
+
+Your assessment must not target other cloud.gov customers, nor perform or simulate denial of service attacks or otherwise violate the [Amazon AWS testing policy](https://aws.amazon.com/security/penetration-testing/).
+
+
+## cloud.gov Pages
```
Comment (jameshochadel) [inline _docs/compliance/pentest.md]: Dangling header?

File: _docs/compliance/domain-standards.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384 (0xC028)
 
 These are false positives. While the CBC cipher modes of operation are being phased out (they are theoretically subject to padding oracle attacks), our cipher implementation is consistent with all relevant guidance, as noted above.
 
-**TLS 1.3**: Where we have not implemented TLS 1.3, it's because the underlying AWS service does not yet support it.
+**TLS 1.3**: Where we have not implemented TLS 1.3, it's because the underlying AWS service does not yet support it. In March, 2023, [AWS announced TLS 1.3](https://aws.amazon.com/about-aws/whats-new/2023/03/application-load-balancer-tls-1-3/) availability for AWS GovCloud, so cloud.gov will pursue implementation by mid-2023.
```
Comment (jameshochadel) [inline _docs/compliance/domain-standards.md:43]: This is fine by me; let's just make sure there's an GH issue for it or it's brought up at PI planning (where we'll then make an issue for it).

**Review by jameshochadel — COMMENTED** [review]
Great clarification Peter, thanks. Just a few minor questions inline.

**Comment by pburkholder** [comment]
The TLS 1.3 work is in my PI plan overview, but not yet a ticket.

**Review by ChrisMcGowan — APPROVED** [review]

**Pull request merged**